test/unit/mdc-menu/menu.foundation.test.js

```diff
@@ -158,9 +157,10 @@ test('handleItemAction item action event inside of a selection group with anothe
   const itemEl = document.createElement('li');
   td.when(mockAdapter.elementContainsClass(itemEl, listClasses.LIST_ITEM_CLASS)).thenReturn(true);
   td.when(mockAdapter.getElementIndex(itemEl)).thenReturn(0);
-  td.when(mockAdapter.getParentElement(itemEl)).thenReturn(itemEl);
   td.when(mockAdapter.elementContainsClass(itemEl, cssClasses.MENU_SELECTION_GROUP)).thenReturn(true);
-  td.when(mockAdapter.getSelectedElementIndex(itemEl)).thenReturn(0);
+  td.when(mockAdapter.isSelectableItemAtIndex(0)).thenReturn(true);
+  td.when(mockAdapter.getSelectedSiblingOfItemAtIndex(0)).thenReturn(0);
```

Comment:
`getSelectedSiblingOfItemAtIndex` will return sibling index that is different index?